lib/shoryuken/environment_loader.rb

```diff
@@ -15,12 +15,12 @@ def initialize(options)
     end
 
     def load
+      Shoryuken.options.merge!(config_file_options)
+      Shoryuken.options.merge!(options)
```

Comment:
This breaks loading the queues from the config when starting with the command-line client.

Basically, the default https://github.com/phstc/shoryuken/blob/master/lib/shoryuken/cli.rb#L115 will cause `options` to always have a `:queues` key and clobber whatever is supplied in the config file.